mainnet-1/specs/movement.json

```diff
@@ -472,12 +475,17 @@
                                 },
                                 "values": [
                                     {
-                                        "latest_distance": 432000
-                                    },
-                                    {
+                                        "extension": "",
```

Comment:
you dont need to add fields that are with default values (empty strings etc..)
also if the verification is the same as aptos you dont need to specify the parse directive, it will take it from there